tests/test_check.py

```diff
@@ -423,3 +423,26 @@ def test_precondition_inline_comment_no_error(caplog) -> None:
     )
 
     assert "WARNING" not in [record.levelname for record in caplog.records]
+
+
+def test_check_all_truncates_messages_when_max_exceeded(capsys):
+    """
+    Test that check_all outputs only the max number of messages set when the max is exceeded.
+    """
+    import python_ta
+
+    python_ta.check_all(
+        # tests/fixtures/unused_imports.py contains 3 unused import errors
+        "tests/fixtures/unused_imports.py",
+        config={
+            "output-format": "pyta-plain",
+            "pyta-error-permission": "no",
```

Comment:
I think you got this from some other tests, but since the default value is `"no"` you don't need to include this option (nor the `pyta-file-permission` option below)